code/__HELPERS/lighting.dm

```diff
@@ -1,5 +1,11 @@
 /// Produces a mutable appearance glued to the [EMISSIVE_PLANE] dyed to be the [EMISSIVE_COLOR].
-/proc/emissive_appearance(icon, icon_state = "", atom/offset_spokesman, layer = FLOAT_LAYER, alpha = 255, appearance_flags = NONE, offset_const, apply_bloom = TRUE)
+/proc/emissive_appearance(icon, icon_state = "", atom/offset_spokesman, layer, alpha = 255, appearance_flags = NONE, offset_const, apply_bloom = TRUE)
+	if (isnull(layer))
+		if(IS_TOPDOWN_PLANE(offset_spokesman.plane))
+			layer = ABOVE_NORMAL_TURF_LAYER + (offset_spokesman.layer - LOW_FLOOR_LAYER + 0.5) * 0.001
```

Comment:
this is weird code and I don't like that we NEED to do this sorta thing. you want to add an offset to ABOVE_NORMAL based off how high up the turf one is yea? would lerp() be better here?